pkg/linter/linter_test.go

```diff
@@ -115,6 +115,7 @@ func TestLinters(t *testing.T) {
 		dirFunc func() string
 		linter  string // NB: Also the package name for this test, but that's weird!
```

Comment:
Maybe change the comment to `"Also used as the fallback package name for the test if unspecified"`